### PR TITLE
Use Sentry releases and upload source maps

### PR DESCRIPTION
**Pull request by edigaryev**
This allows us to resolve issues in Sentry based on the `release` field (even before the new version gets released) and ~~paves a way towards uploading source maps~~ (implemented in https://github.com/cirruslabs/cirrus-ci-web/pull/487/commits/b8ef2ed1187a85819eb350fee9d923b75a328e67).

### CODE REVIEW

**Comment by fkorotkov** [comment]
Should we just use a commit SHA as a release? https://docs.sentry.io/platforms/javascript/configuration/releases/#bind-the-version

**Comment by edigaryev** [comment]
> Should we just use a commit SHA as a release? https://docs.sentry.io/platforms/javascript/configuration/releases/#bind-the-version

IMO version numbers are more human-friendly and will aid in problem solving in the future.

It's more easier to remember the release if it looks like `0.12.0`, rather than `629b18db`, and later compare it with a value that you've previously remembered.

**Comment by fkorotkov** [comment]
We try to deploy every commit so having to tag it every time seems like a burden IMO. Maybe prefix SHA with a date e.g. `20221222@86bf2e7...`?

**Comment by edigaryev** [comment]
> We try to deploy every commit so having to tag it every time seems like a burden IMO.

See https://github.com/cirruslabs/cirrus-ci-web/pull/487/commits/ec41dc777dfa3bba1fb13c41f6d9f3312deb88e9.

**Review by fkorotkov — APPROVED** [review]

File: .cirrus.yml [hunk 1]
```diff
@@ -32,8 +32,19 @@ deploy_docker_builder:
...
     - docker --version
-    - docker build --build-arg SENTRY_DSN=$SENTRY_DSN --cache-from cirrusci/web-front-end:latest --tag cirrusci/web-front-end:latest .
+    - docker build --build-arg CIRRUS_CHANGE_IN_REPO=$CIRRUS_CHANGE_IN_REPO --build-arg SENTRY_DSN=$SENTRY_DSN --cache-from cirrusci/web-front-end:latest --tag cirrusci/web-front-end:latest .
   login_script: docker login --username $DOCKER_USER_NAME --password $DOCKER_PASSWORD
   push_script: docker push cirrusci/web-front-end:latest
+  create_sentry_release_script:
+    - export SENTRY_RELEASE="$CIRRUS_CHANGE_IN_REPO"
+    - sentry-cli releases new $SENTRY_RELEASE
+    - sentry-cli releases set-commits $SENTRY_RELEASE --auto
+    - sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps build/
```
Comment (fkorotkov) [inline .cirrus.yml:49]: Things are gonna get built inside the container. 🤔

**Review by fkorotkov — CHANGES_REQUESTED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (edigaryev) [reply .cirrus.yml:53]: Good catch! See https://github.com/cirruslabs/cirrus-ci-web/pull/487/commits/22cfda60ee1c1f922e842fbde7dc5b0056b68c80 for a potential solution.

**Pull request merged**